eth/vm/base.py

```diff
@@ -289,6 +304,29 @@ def apply_all_transactions(
 
         return result_header, receipts_tuple, computations_tuple
 
+    #
+    # Withdrawals
+    #
+    def apply_withdrawal(
+        self,
+        withdrawal: WithdrawalAPI,
+    ) -> None:
+        self.state.apply_withdrawal(withdrawal)
+
+    def apply_all_withdrawals(
+        self,
+        withdrawals: Sequence[WithdrawalAPI],
+        base_header: BlockHeaderAPI,
+    ) -> None:
+        vm_header = self.get_header()
+        self._validate_header_before_apply(base_header, vm_header)
```

Comment:
see above, do we even need to abstract this out? Leave it just for transactions since that updates the header.